### PR TITLE
Fix warnings on application start

### PR DESCRIPTION
**Pull request by nixerus**
remove various unused imports / variables

### CODE REVIEW

File: src/components/home.js [hunk 1]
```diff
@@ -507,37 +493,6 @@ class home extends Component {
...
-            <Input
-              placeholder="hh:mm:ss"
-              id={`trim-${i}-to`}
-              value={trim.to}
-              onChange={this.onChange}
-            />
-          </div>
-        </Col>
-      </Row>
-    ));
```
Comment (gopavasanth) [inline src/components/home.js]: Hello Netx, Why did you remove this ? Did you test it locally after making all this changes ?

Comment (nixerus) [reply src/components/home.js]: It was an unused variable that was causing a warning on start.
cmd_xJCOK3ke20

**Comment by nixerus** [comment]
Accidentally pushed over this pull request's code, need to close.
